### PR TITLE
feat: Onboard EMBL EBI ChemBL dataset

### PR DESCRIPTION
**Pull request by vijay-google**
## Description

Dataset: ebi_chembl pipeline: chembl_30

## Checklist

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved

### Data Onboarding
- [x] I'm adding or editing a dataset
- [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
- [x] I put all my code inside  `datasets/ebi_chembl` and nothing outside of that directory

### CODE REVIEW

**Comment by vijay-google** [comment]
@adlersantos , @nlarge-google  please review this PR

File: datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cloud-sql-python-connector
+google-cloud-storage
+pandas
+pg8000
```
Comment (nlarge-google) [inline datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/requirements.txt:4]: What is this?

**Review by nlarge-google — CHANGES_REQUESTED** [review]

File: datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+import pathlib
+import time
+
+import pandas as pd
+import pg8000
+import sqlalchemy
+from google.cloud import storage
+from google.cloud.sql.connector import Connector
+
+INT_COLS = [
```
[the remark this answers is not in the document]
Comment (vijay-google) [reply datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/csv_transform.py:29]: Yes it creates duplication. Sure will leave these variables as it is.

File: datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+    def getconn() -> pg8000.dbapi.Connection:
+        conn = connector.connect(
+            instance_connection_name,
+            "pg8000",
+            user=user,
+            password=password,
+            db=database,
+        )
+        return conn
+
```
[the remark this answers is not in the document]
Comment (vijay-google) [reply datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/csv_transform.py]: Yes removed.

File: datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+            upload_file_to_gcs(source_file, target_gcs_bucket, target_gcs_path)
+        else:
+            raise FileNotFoundError(f"{source_file} not found in {os.getcwd()}")
+    logging.info(
+        f"ebi_chembl Dataset chembl_30 pipeline process completed for table(s) -  {tables} at "
+        + str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+    )
+
+
+def get_table_data(
```
[the remark this answers is not in the document]
Comment (vijay-google) [reply datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/csv_transform.py:460]: Modified as per latest changes.

File: datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/requirements.txt (hunk 1 above)
Comment (vijay-google) [reply datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/requirements.txt:4]: For function annotation I needed pg8000 module. Now as per latest changes to main(), removed pg8000 from requirements.txt

**Comment by vijay-google** [comment]
Need to raise another PR.

**Pull request closed without merging**